.github/workflows/pr-command.yml

```diff
@@ -1,31 +1,36 @@
-name: CI build and push (PR)
```

Comment:
I think we should still keep it as `pr-ci.yml` and the name as `CI build and push (PR)`.

Reply:
Maybe something like: `CI build and push (PR) on demand` is also fine.